build.sh

```diff
@@ -1090,8 +1110,12 @@ EOM
                 sh build.sh verify-$target | tee build/build.log | xcpretty -r junit -o build/reports/junit.xml || failed=1
             fi
             if [ "$failed" = "1" ]; then
-                echo "\n\n***\nbuild/build.log\n***\n\n" && cat build/build.log
-                echo "\n\n***\nCoreSimulator.log\n***\n\n" && tail -n2000 ~/Library/Logs/CoreSimulator/CoreSimulator.log
+                echo "\n\n***\nbuild/build.log\n***\n\n" && cat build/build.log || true
+                if [ -f "~/Library/Logs/CoreSimulator/CoreSimulator.log" ]; then
+                echo "\n\n***\nCoreSimulator.log\n***\n\n" && tail -n2000 "~/Library/Logs/CoreSimulator/CoreSimulator.log" || true
```

Comment:
Indentation is really off here. Since we unconditionally create the log file a little earlier, there's no need for the `|| true`. Also, I originally had `tail -n2000` when we didn't empty out the log file on every job, so that can just be `cat`.

There also shouldn't be a need for the `No CoreSimulator.log` echo further down for the same reason.

```bash
if [ "$failed" = "1" ]; then
    echo "\n\n***\nbuild/build.log\n***\n\n" && cat build/build.log || true
    echo "\n\n***\nCoreSimulator.log\n***\n\n" && cat "~/Library/Logs/CoreSimulator/CoreSimulator.log"
    exit 1
fi
```